text/0000-lwc-dom-directive-retorm.md

```diff
@@ -0,0 +1,69 @@
+- Start Date: 2019-10-28
+- RFC PR: (leave this empty)
+- Lightning Web Component Issue: (leave this empty)
+
+# Summary
+
+This RFC describes the formalization of the `lwc:dom` directive in LWC Templates. Specifically, it defines how to lift existing restrictions on `lwc:dom="manual"` and introduces `lwc:dom="synthetic-portal"` as a way to support legacy systems.
+
+# Basic example
+
+This RFC lift some restrictions on the existing implementation and introduces the concept of `synthetic-portal`:
+
+```HTML
+<template>
+
+    <h1>Existing feature (for leaf element only):</h1>
+    <div lwc:dom="manual"></div>
+
+    <h1>Lifting restrictions on existing feature:</h1>
+    <div lwc:dom="manual">
+        <p>can have child elements in template to be combined with manually inserted nodes</p>
+    </div>
+
+    <h1>Introducing new feature (for leaf element only):</h1>
+    <div lwc:dom="synthetic-portal"></div>
```

Comment:
The proposal doesn't provide details about `syntehtic-portal`? What are the intended use cases, and the expected behviors?

Reply:
:) it is WIP, you are not supposed to be reviewing it yet... jajaja... let me finish this today.